pkg/gather/ssh/agent.go

```diff
@@ -0,0 +1,36 @@
+package ssh
+
+import (
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh/agent"
+
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+// newAgent initializes an SSH Agent with the keys.
+// If not keys are provided, it loads all the keys from the user's environment.
```

Comment:
nit: "not keys" -> "no keys"

Reply:
https://github.com/openshift/installer/pull/1822/commits/6c082a8bfda78b0445ef32a21e3effec6275d8c8